### ISSUE TITLE
Checkbox misplaced in iOS and Android.

### ISSUE THREAD

**Issue by murilopereirame** [tutao/tutanota#6215]
The checkbox is misplaced in iOS and Android. This doesn't affect clicking and checking the checkbox but it's difficult to get the state of the checkbox while using screen readers, since the checkbox is behind the text and not together with the checkbox's svg.

## Steps to reproduce
1. Enable TalkBack/VoiceOver
2. Go the the app
3. Try to select the checkbox by tapping the checkbox svg, it'll not be selected
4. Try to select it tapping a little to the right, as the image shows, the checkbox is selected

## iOS
![IMG_0074](https://github.com/tutao/tutanota/assets/34790144/11e07d5e-5275-4284-aa32-a1b989f7472e)

## Android
![Screenshot_20231128-144947](https://github.com/tutao/tutanota/assets/34790144/8864e2ff-e203-4dd4-acdb-5c21461ee0f7)

**Comment by murilopereirame** [comment]
My bad, added the steps to reproduce

**Comment by charlag** [comment]
I don't know why both even need to be visible to screen readers, I find it confusing that there are both there but it's closer to where it should be now at least

**Comment by charlag** [comment]
VoiceOver doesn't see two checkboxes, only TalkBack :shrug:
